### PR TITLE
Unit test color.from sgf prop

### PR DESCRIPTION
**Pull request by RodDeacon**
Created the unit test function for color.FromSGFProp() Issue #178

### CODE REVIEW

File: core/color/color_test.go [hunk 1]
```diff
@@ -41,3 +41,37 @@ func TestOpposite(t *testing.T) {
 		})
 	}
 }
+
+func TestFromSGFProp(t *testing.T) {
```
Comment (artasparks) [inline core/color/color_test.go:46]: I'd recommend trying to make this a table test, now that you have the basics of testing down.

File: core/color/color_test.go [hunk 2]
```diff
@@ -41,3 +41,37 @@ func TestOpposite(t *testing.T) {
...
+
+	out, err = FromSGFProp("W")
+	if err != nil {
+		t.Fatal(err)
+	}
+	if out != Black {
+		t.Errorf("got %v, but wanted %v", out, White)
+	}
+
+	out, err = FromSGFProp("AW")
```
Comment (artasparks) [inline core/color/color_test.go]: Can you add a test for an error-case? In other words, passing in something like "Z" or ""?

File: core/color/color_test.go (hunk 1 above)
Comment (RodDeacon) [reply core/color/color_test.go:46]: OK I will do my best!

File: core/color/color_test.go (hunk 2 above)
Comment (RodDeacon) [reply core/color/color_test.go]: Will do!

**Review by artasparks — APPROVED** [review]
Nice!

**Pull request merged**